### PR TITLE
fix(monitoring): refresh monitoring on content changes

### PR DESCRIPTION
**Pull request by petrjasek**
there is single event `content:update` with info about item, desk
and stage which client can use to figure out what to update in monitoring.

SD-3204 SD-3213

### CODE REVIEW

File: server/apps/content/__init__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        ids[str(item.get('_id', ''))] = 1
+        task = item.get('task', {})
+        if task.get('desk'):
+            desks[str(task.get('desk', ''))] = 1
+        if task.get('stage'):
+            stages[str(task.get('stage', ''))] = 1
+    user = flask.g.get('user', {})
+    push_notification(
+        'content:update',
+        user=str(user.get('_id', '')),
```
Comment (sivakuna-aap) [inline server/apps/content/__init__.py]: can we use `common.py.get_user()` here?

**Comment by sivakuna-aap** [comment]
:+1:

File: server/apps/content/__init__.py (hunk 1 above)
Comment (petrjasek) [inline server/apps/content/__init__.py]: yep I just didn't want to import from archive here, so I will move the func here and import it into archive

Comment (petrjasek) [inline server/apps/content/__init__.py]: actually I will move it into `apps.auth`, imo it should be there

Comment (sivakuna-aap) [inline server/apps/content/__init__.py]: yeah makes sense

**Comment by mdhaman** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Comment by amagdas** [comment]
:+1: Rebase needed

**Pull request merged**